src/js/components/DataTable/Body.js

```diff
@@ -218,10 +250,15 @@ const Body = forwardRef(
               return (
                 <Row
                   key={index}
+                  tableRef={ref}
+                  lastActive={lastActive}
+                  setLastActive={setLastActive}
                   rowRef={rowRef}
                   primaryValue={primaryValue}
                   isSelected={isSelected}
                   isRowExpanded={isRowExpanded}
+                  itemFocus={itemFocus}
```

Comment:
I didn't see where this was used by Row. Do we need to pass it here?

Reply:
Got rid of this here after trimming the unused code as it was incorrectly placed, this was supposed to be put on `StyledDataTableBody`